### PR TITLE
Updates legacy EA docs

### LINKED ISSUE #4190: Update legacy entity analytics docs
## Description 

We need to make a slight change to legacy [host risk score](https://www.elastic.co/guide/en/security/current/host-risk-score.html#upgrade-host-risk-score) and [user risk score](https://www.elastic.co/guide/en/security/current/user-risk-score.html#upgrade-user-risk-score) docs that mention that upgrading those scores as mentioned applies to versions `8.10` and earlier only, as specified per docs, and that if they're upgrading to `8.11`, there's a new risk engine and method of upgrading, which we could add a ref link for when those docs are live.

### PR DESCRIPTION
**Pull request by natasha-moore-elastic**
Resolves #4190.

Updates legacy User & Host risk score docs to specify that the upgrading process described for these scores is applicable only to versions 8.5–8.10. 

Previews:
* [Host risk score](https://security-docs_4193.docs-preview.app.elstc.co/guide/en/security/master/host-risk-score.html)
* [User risk score](https://security-docs_4193.docs-preview.app.elstc.co/guide/en/security/master/user-risk-score.html)

### CODE REVIEW

File: docs/experimental-features/host-risk-score.asciidoc [hunk 1]
```diff
@@ -73,7 +73,7 @@ TIP: If you receive an error message during the installation process, delete the
 [discrete]
 === Upgrade host risk score 
 
-If you previously enabled host risk score and are upgrading to {stack} version 8.5 or later, there'll be an **Upgrade** button for host risk score instead of **Enable**. 
+If you previously enabled host risk score and are upgrading to {stack} versions 8.5–8.10, there'll be an **Upgrade** button for host risk score instead of **Enable**. If you're upgrading to {stack} version 8.11 or newer, you can upgrade to the latest risk scoring engine.
```
Comment (natasha-moore-elastic) [inline docs/experimental-features/host-risk-score.asciidoc:76]: Reference to `<<upgrade-risk-engine>>` to be added once PR #4167 is merged.

File: docs/experimental-features/user-risk-score.asciidoc [hunk 2]
```diff
@@ -69,7 +69,7 @@ TIP: If you receive an error message during the installation process, delete the
 [discrete]
 === Upgrade user risk score 
 
-If you previously enabled user risk score and are upgrading to {stack} version 8.5 or later, there'll be an **Upgrade** for user risk score instead of **Enable**.  
+If you previously enabled user risk score and are upgrading to {stack} versions 8.5–8.10, there'll be an **Upgrade** for user risk score instead of **Enable**. If you're upgrading to {stack} version 8.11 or newer, you can upgrade to the latest risk scoring engine.
```
Comment (natasha-moore-elastic) [inline docs/experimental-features/user-risk-score.asciidoc:72]: Reference to <<upgrade-risk-engine>> to be added once PR https://github.com/elastic/security-docs/pull/4167 is merged.

File: docs/experimental-features/host-risk-score.asciidoc (hunk 1 above)
Comment (joepeeples) [inline docs/experimental-features/host-risk-score.asciidoc:76]: I'm kinda confused: it sounds like regardless of the user's version, they can upgrade (we say there's an Upgrade button, or just that they can upgrade). It's not clear why we're distinguishing 8.5-8.10 from 8.11+ here.

File: docs/experimental-features/user-risk-score.asciidoc (hunk 2 above)
Comment (joepeeples) [reply docs/experimental-features/user-risk-score.asciidoc:72]: Same comment as above.

File: docs/experimental-features/host-risk-score.asciidoc (hunk 1 above)
Comment (natasha-moore-elastic) [reply docs/experimental-features/host-risk-score.asciidoc:76]: My understanding is that if they're upgrading from an earlier version to 8.5-8.10, they will still be using the 'original' (i.e. old) risk engine. Whereas if they're upgrading to 8.11, they can upgrade to the 'latest' (new) risk engine, for which the upgrade steps are different (or they can continue to run the old one if they want).
@SourinPaul, could you confirm if this is correct?

Comment (SourinPaul) [reply docs/experimental-features/host-risk-score.asciidoc:76]: > My understanding is that if they're upgrading from an earlier version to 8.5-8.10, they will still be using the 'original' (i.e. old) risk engine. Whereas if they're upgrading to 8.11, they can upgrade to the 'latest' (new) risk engine, for which the upgrade steps are different (or they can continue to run the old one if they want).

@natasha-moore-elastic, your understanding above is correct.

Comment (SourinPaul) [reply docs/experimental-features/host-risk-score.asciidoc:76]: @natasha-moore-elastic 💡 Will listing the two scenarios (stack version 8.5- 8.10 & stack version 8.11) as a bulleted list, improve the readability/ comprehension?

**Review by SourinPaul — APPROVED** [review]

File: docs/experimental-features/host-risk-score.asciidoc (hunk 1 above)
Comment (joepeeples) [reply docs/experimental-features/host-risk-score.asciidoc:76]: Thanks for explaining and confirming, @natasha-moore-elastic & @SourinPaul. 

I think part of the problem is we're using 8.11 docs to document 8.5-8.10 functionality & behavior. For 8.11 docs, we don't need to explain what happens if a user upgrades _within the 8.5-8.10 range_, which is what that first sentence is talking about. We only need to explain what happens if they upgrade _to 8.11 or later_.

If there's different behavior depending on which version they're upgrading _from_, a bulleted list would be a good way to clearly distinguish the two scenarios.

Comment (jmikell821) [reply docs/experimental-features/host-risk-score.asciidoc:76]: I think in the interest of time, let's focus on directions for upgrading to `8.11` or newer, then we'll tackle how to refer to upgrading in previous doc versions.

**Pull request merged**